libopae/src/event.c

```diff
@@ -352,6 +352,8 @@ static fpga_result driver_register_event(fpga_handle handle,
 		} else if (objtype == FPGA_ACCELERATOR) {
 			return send_port_event_request(handle, event_handle, FPGA_IRQ_ASSIGN);
 		}
+		FPGA_ERR("Invalid objtype: %d", objtype);
+		return FPGA_EXCEPTION;
```

Comment:
@tswhison should we try to avoid multiple return statements in a function by setting the "res = FPGA_EXCEPTION in default case and returning the 'res' variable at the end of this function?

Reply:
Not sure that would address the warning here.  The warning was that the switch case was implicitly fall-through.  So we needed either a break or return to satisfy the compiler.

Reply:
@tswhison This is not related to the warning here. Let's discuss the coding style in the future.